src/models/miner.model.js

```diff
@@ -10,33 +10,28 @@ const DiamondApi = require('src/api/diamond.api.js');
 const diamondApi = new DiamondApi();
 const cache = require('src/util/cache.js');
 
-class MinerModel {
-
-    address = null;
-    id = null;
-    name = null;
-    difficulty = config.get('pool:difficulty');
-    port=config.get('pool:port');
-    shareTimeRing = cryptonoteUtils.ringBuffer(16);
-    
+class MinerModel {    
     /**
      * Serialize from JSON or JWT token into a Miner instance
      * @param {object} data 
      */
-    async constructor(data) {
+    constructor(data) {
         
         this.address = data.address;
         this.name = data.name;
         this.id = data.accountId;
+        this.difficulty = config.get('pool:difficulty')
```

Comment:
Do we need to map difficulty to miner? Or do you mean the hashrate that the miner is providing

Reply:
That's a good suggestion, this is copied from the other pool, I guess the idea is that every miner has a start difficulty based on the port they connected to, and ca potentially update their difficulty based on their hashrate, 
🤷‍♂ IDK, what do you think, should be keep difficulty uniform for all miners on a specific port? @andyjung-dev

Reply:
Resolved, this is removed